### PR TITLE
bpo-35450: reflect in docs that venv module is not always creating a …

### PR DESCRIPTION
**Pull request by mkkot**
…copy of python binary

https://bugs.python.org/issue35450

### CODE REVIEW

File: Doc/using/venv-create.inc [hunk 1]
```diff
@@ -7,9 +7,10 @@ Running this command creates the target directory (creating any parent
 directories that don't exist already) and places a ``pyvenv.cfg`` file in it
 with a ``home`` key pointing to the Python installation from which the command
 was run.  It also creates a ``bin`` (or ``Scripts`` on Windows) subdirectory
-containing a copy of the ``python`` binary (or binaries, in the case of
-Windows).  It also creates an (initially empty) ``lib/pythonX.Y/site-packages``
-subdirectory (on Windows, this is ``Lib\site-packages``). If an existing
+containing a copy/symlink of the Python binary/binaries (as appropriate by the
```
Comment (brettcannon) [inline Doc/using/venv-create.inc]:
```suggestion
containing a copy/symlink of the Python binary/binaries (as appropriate for the
```

**Review by brettcannon — CHANGES_REQUESTED** [review]

**Comment by brettcannon** [comment]
@mkkot only one minor grammatical change to make. Otherwise I'm just waiting for your CLA to be processed and then I can merge the change!

**Comment by mkkot** [comment]
Sure, please merge. What about backporting, who is dealing with this?

**Comment by brettcannon** [comment]
@mkkot our magical bots handle backports 😄  (that's what the labels actually trigger).

**Comment by brettcannon** [comment]
@mkkot I can't merge until you either accept the suggested fix or click the checkbox to let maintainers modify your branch (I just tried and GitHub says I can't do it).

**Comment by mkkot** [comment]
@brettcannon I resolved the conversation, hope now you can merge.

**Review by brettcannon — APPROVED** [review]

**Comment by brettcannon** [comment]
@mkkot I actually wanted the suggested change to get merged, but it turned out I had the appropriate permissions to edit the file, so I just went ahead and did that.

Thanks for the PR!

**Pull request merged**